src/main/java/CookingConverter/CookingMeasurementConverter.java

```diff
@@ -0,0 +1,40 @@
+package CookingConverter;
+
+import static CookingConverter.Cooking.Units.*;
+
+public class CookingMeasurementConverter {
+    private CookingMeasurementConverter() {
+    }
+
+    private static float getMeasurementRatio(String unit) {
+        switch (unit) {
+            case TSP:
+                return Cooking.Amounts.TSP;
```

Comment:
could move this function to amounts, won't need static class ref. Or could use static import for the amounts too just like you did for units